pkg/services/rpcsrv/server.go

```diff
@@ -2097,7 +2090,7 @@ func (s *Server) getInvokeContractVerifyParams(reqParams params.Params) (util.Ui
 	if len(reqParams) > 1 {
 		args, err := reqParams[1].GetArray() // second `invokecontractverify` parameter is an array of arguments for `verify` method
 		if err != nil {
-			return util.Uint160{}, nil, nil, neorpc.WrapErrorWithData(neorpc.ErrInvalidParams, err.Error())
+			return util.Uint160{}, nil, nil, neorpc.WrapErrorWithData(neorpc.ErrInvalidVerificationFunction, err.Error())
```

Comment:
It's `ErrInvalidParams`. `ErrInvalidVerificationFunction` should be returned from `invokecontractverify` in case if there's no `verify` method in the contract. You have to write some code that distinguishes the following error:
https://github.com/nspcc-dev/neo-go/blob/e7a83cbef95469e8b88627f98ff3fec8445e19ba/pkg/core/blockchain.go#L2805

Reply:
Ok, done